src/setting/settings.ts

```diff
@@ -64,6 +64,22 @@ export class ObsidianGitSettingsTab extends PluginSettingTab {
 
         let setting: Setting;
         if (gitReady) {
+            new Setting(containerEl).setName("Activation").setHeading();
```

Comment:
I don't think a setting is needed for this. (I want to keep the amount of settings rather low) I guess a command is enough.